scrapers/scrape_zg.py

```diff
@@ -1,21 +1,46 @@
 #!/usr/bin/env python3
 
+import re
+from io import StringIO
+from urllib.parse import urlparse
+from bs4 import BeautifulSoup
+import pandas as pd
 import scrape_common as sc
 
 print('ZG')
 d = sc.download('https://www.zg.ch/behoerden/gesundheitsdirektion/amt-fuer-gesundheit/corona')
 sc.timestamp()
+soup = BeautifulSoup(d, 'html.parser')
 d = sc.filter(r'Infizierte Personen|Genesene Personen|Verstorbene Personen|Stand:', d)
+print('Date and time:', sc.find(r'Stand:? ([^<]+ Uhr)<', d))
 
-# 2020-03-23
-"""
+detailed_stats = soup.find("a", text=re.compile("Detaillierte Statistik"))
+if detailed_stats:
+    d_detailed_stats = sc.download(detailed_stats['href'])
+    d_detailed_stats_comps = urlparse(detailed_stats['href'])
+    csv_path = sc.find(r'csv_path:"([^"]+)"', d_detailed_stats)
+    csv_url = f"{d_detailed_stats_comps.scheme}://{d_detailed_stats_comps.netloc}{csv_path}"
+    d_csv = sc.download(csv_url)
+    df = pd.read_csv(StringIO(d_csv), sep=",", parse_dates=['Datum'], dayfirst=True)
```

Comment:
Wouldn't using just `csv` (standard Python module) module be easier than pandas? I don't have pandas even installed.

Reply:
I found pandas just quicker for me to implement but give me a couple of minutes. I'll convert it to csv

Reply:
Well, I only now noticed that the data is transposed. So, it might be easier with pandas. Whatever looks cleaner in code.

Reply:
Pandas read_csv function is fantastic for reading and parsing csv files. It's a one-liner ;)

Reply:
Converted it to CSV ... not much more complicated in this case

Reply:
Ok. Thanks.